packages/client/src/upload.ts

```diff
@@ -82,6 +82,9 @@ export async function* upload(
   const defaultAgent = apiUrl?.startsWith('https://')
     ? new https.Agent({ keepAlive: true })
     : new http.Agent({ keepAlive: true });
+  const abortController = new AbortController();
+  // default max is 10, but abort controller may have 50 connections at once
+  events.setMaxListeners(50);
```

Comment:
This defines global behavior for the current Node.js process. I don't think it should live as a side effect of this upload logic. Can we move it to the entrypoint of CLI or the entrypoint of a command? This will make it clearer that this is happening at a higher level.